.github/workflows/push.yaml

```diff
@@ -0,0 +1,74 @@
+name: push
+
+env:
+  AWS_REGION: us-east-1
+  AWS_ROLE_DURATION: 3600
+  # role generated from https://github.com/Sceptre/sceptre-aws/blob/master/config/prod/gh-oidc-sceptre-tests.yaml
+  AWS_ROLE: arn:aws:iam::743644221192:role/gh-oidc-sceptre-tests
+
+on:
+  workflow_run:
+    workflows:
+      - check
+    types:
+      - completed
+  push:
+    branches:
+      - '*'
```

Comment:
this workflow isn't getting triggered from my testing.  i'm not sure whether it's because this snippet isn't working or whether this workflow needs to be bootstrapped before in GH actions before it will work.

Reply:
i've tested and verified it should be..
```
on:
  workflow_run:
    workflows:
      - check
    types:
      - completed
    branches:
      - master
```